src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java

```diff
@@ -135,6 +137,15 @@ public AdyenConfigProperties(final Properties properties, final String currentRe
 
         this.merchantAccounts = properties.getProperty(PROPERTY_PREFIX + "merchantAccount");
         refillMap(countryToMerchantAccountMap, merchantAccounts);
+        if(this.countryToMerchantAccountMap.containsKey(FALL_BACK_MERCHANT_ACCOUNT_KEY))
```

Comment:
Could you fix the style (braces placement, spacing, etc.)?

Reply:
Done in 
638f8a8